testutil/environment/environment.go

```diff
@@ -225,3 +225,8 @@ func EnsureFrozenImagesLinux(testEnv *Execution) error {
 func (e *Execution) GitHubActions() bool {
 	return os.Getenv("GITHUB_ACTIONS") != ""
 }
+
+// UsesSnapshotter is true if the daemon is configured to use the containerd snapshotter.
+func (e *Execution) UsesSnapshotter() bool {
+	return os.Getenv("TEST_INTEGRATION_USE_SNAPSHOTTER") != ""
+}
```

Comment:
Heh. I should probably have split my commit from https://github.com/moby/moby/pull/45232/commits/641cfe4b24122bfc7b3a9cc8c4231f51cb30fbc3 😂 

```go
// UsingSnapshotter returns whether containerd snapshotters are used for the
// tests by checking if the "TEST_INTEGRATION_USE_SNAPSHOTTER" is set to a
// non-empty value.
func (e *Execution) UsingSnapshotter() bool {
	return os.Getenv("TEST_INTEGRATION_USE_SNAPSHOTTER") != ""
}
```

Reply:
Yeah I was surprised that we don't actually have it yet 😄 
Which PR is it part of?

Reply:
Oh that's part of your https://github.com/moby/moby/pull/45232 (which I rebased / updated a few times) - I created that commit, but then didn't have any tests to use it for, so I only left that commit in that branch.

I think either one is fine; mine does exactly the same, just a different name and GoDoc